source/_cookbook/track_battery_level.markdown

```diff
@@ -41,3 +45,19 @@ sensor:
     device_class: battery
 ```
 {% endraw %}
+
+#### HTTP
+
+If you have configured Owntracks to send reports to your Homeassistant instance via HTTP you can use a template sensor. 
+Replace deviceid with the set Device ID in Owntracks.
+
+{% raw %}
+```yaml
+sensor:
+- platform: template
+    sensors:
+      your_battery_sensor_name:
+        value_template: '{{ states.device_tracker.deviceid.attributes.battery_level }}'
```

Comment:
We should not retrieve attributes like that, please use `state_attr('device_tracker.deviceid', 'battery_level')` instead.
More information: <https://www.home-assistant.io/docs/configuration/templating/#states>